main-page.css

```diff
@@ -0,0 +1,213 @@
+body {
+    margin: 0;
+}
+.right-pannel {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    border-radius: 10px 0 0 10px;
+    width: 80px;
+    height: 250px;
+    background-color: white;
+    position: fixed;
+    float: right;
+    top: 40%;
+    right: 0;
+}
+.icons {
+    width: 50px;
+    height: 50px;
+}
+.icons:first-child {
+    margin-top: 10px;
+}
+.icons:last-child {
+    margin-bottom: 10px;
+}
+
+.main-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+}
+.first-screen {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    background-color: rgb(0, 0, 0);
+    color: white;
+}
+.header-part {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 80%;
+    height: 150px;
+    border-bottom: 2px solid white;
+}
+.profile-photo {
+    width: 80px;
+    height: 80px;
+    border-radius: 50%;
+    border: 2px solid white;
+}
+.left-part h1{
+    margin-left: 15px;
+}
+.left-part {
+    display: flex;
+    color: white;
+    margin-left: 30px;
+}
+.nav {
+    display: flex;
+    justify-content: end;
+    list-style-type: none;
+    font-size: 30px;
+    margin-right: 30px;
+    padding: 0;
```

Comment:
Let's create 'reset.css', and avoid handle reset. This file will reset all default browser values. U can google this file.

Reply:
fixed